qemu/tests/cfg/boot_from_virtiofs.cfg

```diff
@@ -11,6 +11,12 @@
     fs_passwd = virtiofs
     set_passwd_cmd = "echo ${fs_passwd} | passwd --stdin root ; exit"
     images = ""
+    ovmf:
+        images += "stg"
+        image_size_stg = 10G
```

Comment:
Could you change the value a bit little as it's not used at all.
And could you add some notes that it's just a workaround.
And the patch title seems a little weird, how about 'xxxx: support ovmf with a workaround '

Reply:
updated.